articles/app-service/configure-authentication-provider-aad.md

```diff
@@ -50,8 +48,10 @@ You can also provide configuration settings manually. This is the preferred solu
 4. In a few seconds, you should see the new app registration you just created.
 5. Once the app registration has been added, click on the app registration name, click on **Settings** at the top, then click on **Properties** 
 6. In the **App ID URI** box, paste in the Application URL (from step 1), also in the **Home Page URL** paste in the Application URL (from step 1) as well, then click **Save**
-7. Now click on the **Reply URLs**, edit the **Reply URL**, paste in the Application URL (from step 1), then appended to the end of the URL, */.auth/login/aad/callback* (For example, `https://contoso.azurewebsites.net/.auth/login/aad/callback`). Click **Save**.   
-8.  At this point, copy the **Application ID** for the app. Keep it for later use. You will need it to configure your App Service app.
+7. Now click on the **Reply URLs**, edit the **Reply URL**, paste in the Application URL (from step 1), then appended to the end of the URL, */.auth/login/aad/callback* (For example, `https://contoso.azurewebsites.net/.auth/login/aad/callback`). Click **Save**.
+    * Consider using the same app registration for multiple environments when using Slots of your Web App service by simply adding more **Reply URLs**.
```

Comment:
I think we should use the note syntax (a custom markdown extension we have described [here](https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown#ops-custom-markdown-extensions)) rather than a sub-bullet here. I'd also change the consider language to just describe the ability and combine the two points since they are connected.

How about the following (and thanks again for your flexibility and iteration):

\> [!NOTE]
\> You can use the same app registration for multiple domains by adding additional **Reply URLs**. Make sure to model each App Service instance with its own registration so it has its own permisions and consent. Also consider using separate app registrations for separate site slots. This is to avoid permissions being shared between environments, so that a bug in new code you are testing does not affect production.